decidim-core/app/presenters/decidim/user_presenter.rb

```diff
@@ -13,6 +13,8 @@ class UserPresenter < SimpleDelegator
     # nickname presented in a twitter-like style
     #
     def nickname
+      return "@blocked_user" if __getobj__.blocked?
```

Comment:
Could we just return an empty string or even `nil` in this situation?

I think this might be otherwise confusing if we don't translate this (and what if someone wants that particular nickname?).

Also the related spec needs to be changed accordingly.

Reply:
It should be okay now !